assets/src/edit-story/output/utils/getGoogleFontURL.js

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Given a list of Google fonts, returns a URL to embed them.
+ *
+ * @see https://developers.google.com/fonts/docs/css2
```

Comment:
It's really a shame this doesn't support a different character subset per font - then it would be perfect for our font dropdown! But I guess we want to write the font name in the dropdown, so it will vary per font.

Reply:
There's `subset=menu` option. E.g. https://fonts.googleapis.com/css?family=Roboto%7COpen+Sans&subset=menu. (I also updated the font preview issue with this information.)